src/content/docs/en/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -0,0 +1,379 @@
+---
+title: Migrating from Create React App
+description: Tips for migrating an existing Create React App project to Astro
+type: migration
+stub: true
+framework: Create React App
+i18nReady: true
+---
+import AstroJSXTabs from '~/components/tabs/AstroJSXTabs.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+Astro's [React integration](/en/guides/integrations-guide/react/) provides support for [using React components inside Astro components](/en/core-concepts/framework-components/), including entire React apps like Create React App!
+
+```astro title="src/pages/index.astro"
+---
+// Import your root App component
+import App from '../cra-project/App.jsx'
+---
+<!-- Use a client directive to load your app -->
+<App client:load/>
+```
+
+Many apps will "just work" as full React apps when you add them directly to your Astro project with the React integration installed. This is a great way to get your project up and running immediately, and keep your app functional while you migrate to Astro. 
+
+Over time, you can convert your structure piece-by-piece to a combination of `.astro` and `.jsx` components. You will probably discover you need fewer React components than you think!
+
+Here are some key concepts and migration strategies to help you get started. Use the rest of our docs and our [Discord community](https://astro.build/chat) to keep going!
+
+## Key Similarities between Create React App and Astro
+
+- The [syntax of `.astro` files is similar to JSX](/en/core-concepts/astro-syntax/#differences-between-astro-and-jsx). Writing Astro should feel familiar.
+
+- Astro uses file-based routing, and [allows specially named pages to create dynamic routes](/en/core-concepts/routing/#dynamic-routes).
+
+- Astro is [component-based](/en/core-concepts/astro-components/), and your markup structure will be similar before and after your migration.
+
+- Astro has [official integrations for React, Preact, and Solid](/en/guides/integrations-guide/react/) so you can use your existing JSX components. Note that in Astro, these files **must** have a `.jsx` or `.tsx` extension.
+
+- Astro has support for [installing NPM packages](/en/guides/imports/#npm-packages), including React libraries. Many of your existing dependencies will work in Astro.
+
+## Key Differences between Create React App and Astro
+
+When you rebuild your Create React App site in Astro, you will notice some important differences:
```

Comment:
Could we possibly introduce the CRA acronym here or is it more appropriate later on? 

I was also curious of what your thoughts would be having `(CRA)` in title? That way you save the writing in parts such as `Key Difference between CRA and Astro`, as an example.

Reply:
Yes, those would make for nicer side bar headings. I could see putting it in the title, and maybe even also in the opening line? Let's try both places and then use the acronym anywhere the long name absolutely feels too long (esp. headings, for sidebar real estate.)